yt_dlp/extractor/ixigua.py

```diff
@@ -54,7 +55,7 @@ def _get_video_format(self, video_type, video_type_json):
                 'fps': int_or_none(video.get('fps')),
                 'vcodec': video.get('codec_type'),
                 'format_id': str(video.get('quality_type')),
-                'ext': 'mp4' if parse_qs(video_url).get('mime_type')[0] == 'video_mp4' else None,
+                'ext': 'mp4' if parse_qs(video_url).get('mime_type')[0] == 'video_mp4' else 'mp4', 
```

Comment:
```suggestion
                'ext': 'mp4', 
```

It's always MP4.

Reply:
Actually i not sure to make this always mp4 as they give audio and video at different link

Reply:
But your code can only get mp4 (` mp4 if X else mp4`)

Reply:
I know that, i make like because i assume the dash link contain full url not separated video and audio

Reply:
For the audio, i think i will make separate ext.